week1/project/hackyourweather/src/components/Weather.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import cities from "./city-weather.json";
+
+function Weather() {
+  const cityWeather = cities.map((city) => (
```

Comment:
destructuring is your friend, making use of it, will make your code readable and cleaner.

```suggestion
  const cityWeather = cities.map(({ name, sys, weather, main, coord }) => (
```

**caution: breaking changes.**